### PR TITLE
Fix build documentation for rbx.

### PR DESCRIPTION
**Pull request by ad-m**
To generate documentation for ```rbx``` you need authenticate & next to:

```
SCOPE_NAME="rbx" NODE_ENV="development" npm run docs
```

### CODE REVIEW

**Comment by ad-m** [comment]
@fredericosilva , there is any npm-native way to set ```NODE_ENV``` variable?

File: bin/login/index.js [hunk 1]
```diff
@@ -39,9 +40,14 @@ const handler = args => {
...
     }
 
+    const p_config = args.helpers.api.get('/cli');
+
     return p
-        .then(() => logger('info', 'You successfully logged and stored your apiKey in config file'))
-        .catch(e => {
+        .then(() => {
+            logger('info', 'You successfully logged and stored your apiKey in config file');
+            p_config.then(cli_config => config.set('cli', cli_config));
```
Comment (fredericosilva) [inline bin/login/index.js]: promise chain is broken, a return is missing

File: docs/vault.md [hunk 2]
```diff
@@ -416,7 +416,7 @@ Connect to Vault using SSH
 ### Example
 
 ```bash
-sh1 vault ssh --vault my-vault
+h1 vault ssh --vault my-vault
```
Comment (fredericosilva) [inline docs/vault.md]: {{scope}} missing?

File: scripts/print_docs.js [hunk 3]
```diff
@@ -45,6 +44,7 @@ const entry_filename = (entry) => `${entry.name}.md`;
 const writeCommandTOC = (stream, entry, prefix, depth) => {
     if (!entry.children) return;
     entry.children.forEach(entry => {
+        if (entry.createOptions.skipDocumentation) return;
```
Comment (fredericosilva) [inline scripts/print_docs.js]: maybe filter before?

File: docs/vault.md (hunk 2 above)
Comment (ad-m) [reply docs/vault.md]: No, it's builded documentation. See that fix https://github.com/hyperonecom/h1-cli/pull/137/files/a6b263b2785d9e8c3e77e6a03dde5fa3c3144d8e#diff-c7b006c46344ce3aabf1b9dfb58e75e5 .

File: scripts/print_docs.js (hunk 3 above)
Comment (ad-m) [reply scripts/print_docs.js]: Good idea.

File: docs/vault.md (hunk 2 above)
Comment (fredericosilva) [reply docs/vault.md]: Upss, I miss read the location

**Pull request merged**